### PR TITLE
feat(Twitter): Add support for Twitter timelines

### PR DESCRIPTION
**Pull request by acagastya**
**What**: Implemented Twitter timelines as mentioned in #97 

**Why**: 🤨

**How**: Works the same way twitter moments or events would work -- so.

**Checklist**:

- [ ] Documentation N/A -- Updated Twitter.js which already has docs
- [x] Tests
- [x] Ready to be merged
      
You *can* merge it after reviewing.

### CODE REVIEW

File: src/transformers/Twitter.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export const shouldTransform = (url) => {
   return (
     ['twitter.com', 'www.twitter.com'].includes(host) &&
     (pathname.includes('/status/') ||
-      (includesSomeOfArray(pathname, ['/events/', '/moments/']) &&
+      (includesSomeOfArray(pathname, ['/events/', '/moments/', '/timelines']) &&
```
Comment (MichaelDeBoey) [inline src/transformers/Twitter.js:9]:
```suggestion
      (includesSomeOfArray(pathname, ['/events/', '/moments/', '/timelines/']) &&
```

File: src/__tests__/transformers/Twitter.js [hunk 2]
```diff
@@ -56,6 +60,10 @@ cases(
       url: 'https://twitter.com/i/moments/edit/994601867987619840',
       valid: false,
     },
+    'timeline edit url': {
+      url: 'https://twitter.com/wesbos/timelines/edit/1189618481672667136',
+      valid: false,
+    },
```
[the remark this answers is not in the document]
Comment (acagastya) [reply src/__tests__/transformers/Twitter.js:68]: I don't know if it is -- I had never seen a timeline before I saw the issue -- but I guess the author of the time line could -- if not, we can just remove it.

[the remark this answers is not in the document]
Comment (MichaelDeBoey) [reply src/__tests__/transformers/Twitter.js:68]:
```suggestion
```

File: src/__tests__/transformers/Twitter.js [hunk 3]
```diff
@@ -131,10 +164,14 @@ test('Plugin can transform Twitter links', async () => {
...
     <https://this-is-not-twitter.com/foobar/status/123>
 
+    <https://this-is-not-twitter.com/foobar/timelines/123>
+
     <https://twitter.com/MichaelDeBoey93>
 
     <https://twitter.com/i/moments/edit/994601867987619840>
 
+    <https://twitter.com/wesbos/timelines/edit/1189618481672667136>
+
```
Comment (MichaelDeBoey) [inline src/__tests__/transformers/Twitter.js:174]:
```suggestion
```

File: src/__tests__/transformers/__fixtures__/Twitter.md [hunk 4]
```diff
@@ -8,10 +8,14 @@ https://this-is-not-twitter.com/i/moments/123
...
 https://this-is-not-twitter.com/foobar/status/123
 
+https://this-is-not-twitter.com/foobar/timelines/123
+
 https://twitter.com/MichaelDeBoey93
 
 https://twitter.com/i/moments/edit/994601867987619840
 
+https://twitter.com/wesbos/timelines/edit/1189618481672667136
+
```
Comment (MichaelDeBoey) [inline src/__tests__/transformers/__fixtures__/Twitter.md:18]:
```suggestion
```

**Comment by acagastya** [comment]
You are welcome, @MichaelDeBoey.  :-)

**Pull request merged**